core/modules/replica/tools/qserv-replica-job-fixup.cc

```diff
@@ -0,0 +1,226 @@
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+/// qserv-replica-job-fixup.cc is a single job Controller application
+/// which is meant to run the corresponding job.
+
+// System headers
+#include <atomic>
+#include <iomanip>
+#include <iostream>
+#include <map>
+#include <set>
+#include <stdexcept>
+#include <string>
+#include <vector>
+
+// Qserv headers
+#include "proto/replication.pb.h"
+#include "replica/Configuration.h"
+#include "replica/Controller.h"
+#include "replica/FixUpJob.h"
+#include "replica/ServiceProvider.h"
+#include "util/BlockPost.h"
+#include "util/CmdLineParser.h"
+
+using namespace lsst::qserv;
+
+namespace {
+
+// Command line parameters
+
+std::string databaseFamily;
+std::string configUrl;
+bool        progressReport;
+bool        errorReport;
+bool        chunkLocksReport;
+
+/// Run the test
+bool test() {
+
+    try {
+
+        ///////////////////////////////////////////////////////////////////////
+        // Start the controller in its own thread before injecting any requests
+        // Note that omFinish callbak which are activated upon a completion
+        // of the requsts will be run in that Controller's thread.
+
+        replica::ServiceProvider::Ptr const provider   = replica::ServiceProvider::create(configUrl);
+        replica::Controller::Ptr      const controller = replica::Controller::create(provider);
+
+        controller->run();
+
+        ////////////////////////////////////////
+        // Find all replicas accross all workers
+
+        std::atomic<bool> finished{false};
+        auto job = replica::FixUpJob::create(
+            databaseFamily,
+            controller,
+            std::string(),
+            [&finished] (replica::FixUpJob::Ptr job) {
+                finished = true;
+            }
+        );
+        job->start();
+
+        util::BlockPost blockPost(1000,2000);
+        while (not finished) {
+            blockPost.wait();
+        }
+
+        //////////////////////////////
+        // Analyse and display results
+
+        replica::FixUpJobResult const& replicaData = job->getReplicaData();
+
+        std::cout
+            << "\n"
+            << "WORKERS:";
+        for (auto const& worker: provider->config()->workers()) {
+            std::cout << " " << worker;
```

Comment:
how come the other `std::cout`s get their own line, but this one has to share? it wants its contract renegotiated.

Reply:
fixed ... for the sake of consistency